app/lib/components/term.js

```diff
@@ -149,6 +160,10 @@ export default class Term extends Component {
     if (this.props.customCSS !== nextProps.customCSS) {
       this.term.prefs_.set('user-css', this.getStylesheet(nextProps.customCSS));
     }
+
+    if (this.devicePixelRatio !== window.devicePixelRatio) {
+      this.term.prefs_.set('font-smoothing', this.getFontSmoothing());
```

Comment:
Can `window.devicePixelRatio` change? I think we probably need to do it only once in the ctor?